app/screens/about/about.js

```diff
@@ -206,6 +206,16 @@ export default class About extends PureComponent {
                         />
                         {licensee}
                         {learnMore}
+                        {!DeviceInfo.getBundleId().includes('mattermost') &&
```

Comment:
Don’t know if that’s enough cause someone can set the bundle id to something like `com.company.mattermost` and this won’t show I would be more confident if we use both of our bundle ids